string frame_id
string personId    #Unique id assigned for each person
float32 distance_travelled     #Distance travelled by the person 	
float32 distancefromHallwayL   #Distance between the person and Left Hallway
float32 distancefromHallwayR   #Distance between the person and Right Hallway
float32 robotHallwayR 	       #Distance between the robot and Right Hallway	
float32 robotHallwayL 	       #Distance between the robot and Left Hallway	  
